docs/user_guide/build_tb.rst

```diff
@@ -0,0 +1,159 @@
+
+**Please note that ADI only provides the source files necessary to create and 
+build the designs. This means that you are responsible for modifying and
+building these projects.**
+
+Here, we are giving you a quick rundown on how we build things. That said,
+the steps below are **not a recommendation**, but a suggestion.
+How you want to build these projects is entirely up to you.
+The only catch is that if you run into problems, you have to resolve them independently.
+
+The build process depends on certain software and tools, which you could use in
+many ways. We use **command line** and mostly **Linux systems**. On Windows, we
+use **Cygwin**.
+
+Set up the Testbenches repository
+-------------------------------------------------------------------------------
+
+.. caution::
+
+   Before building any testbench, you **must**:
+
+   #. Clone the :git-hdl:`HDL repository </>` 
+   
+   #. Check the Vivado version needed by entering the
+      :git-hdl:`hdl/scripts/adi_env.tcl <scripts/adi_env.tcl>` file. If you do
+      not want to use that (although **we strongly advise you to use it**)
+      then you have the alternative of setting ``export ADI_IGNORE_VERSION_CHECK=1``
+      before building the project. Otherwise your project will fail.
+
+   #. :ref-hdl:`Set up the HDL repository <build_hdl>`
+
+
+The :git-testbenches:`Testbenches repository </>` has to be cloned under the
+:git-hdl:`HDL repository </>` as follows:
+
+.. code-block:: bash
+
+   [~] cd hdl
+   [~] git clone git@github.com:analogdevicesinc/testbenches.git
+
+The above command clones the **default** branch, which is the **main** for
+Testbenches. The **main** branch always points to the latest stable release
+branch, but it also has features **that are not fully tested**. If you
+want to switch to any other branch you need to checkout that branch:
+
+.. code-block:: bash
+
+   [~] cd testbenches/
+   [~] git status
+   [~] git checkout 2022_r2      
+
+Building a test bench
+-------------------------------------------------------------------------------
+
+.. caution::
+
+   Before building any test bench, you must have the environment prepared and the
+   proper tools.
+
+The way of building a test bench in Cygwin and WSL is almost the same.
+In this example, it is building the **AD7616** test bench.
+
+.. code-block:: bash
+
+   cd ad7616
+   make
+
+The ``make`` builds all the libraries first and then builds the test bench.
+This assumes that you have the tools and licenses set up correctly. If
+you don't get to the last line, the make failed to build one or more
+targets: it could be a library component or the project itself. There is
+nothing you can gather from the ``make`` output (other than which one
+failed). The actual information about the failure is in a log file inside
+the project directory.
+
+Also, there's an option to use ``make`` using GUI, so that at the end of the
+build it will launch Vivado and start the simulation.
```

Comment:
Also, there's an option to use ``make`` using GUI, so that at the end of the
build it will launch Vivado and start the simulation with the waveform viewer
started as well.